chain/substreams/examples/README.md

```diff
@@ -0,0 +1,13 @@
+## Substreams example
```

Comment:
I think all example code should be added to https://github.com/graphprotocol/example-subgraph or a similar repo for substreams @azf20 thoughts?